tests/python/quantization/test_quantization.py

```diff
@@ -353,6 +378,11 @@ def get_fp32_sym():
 
 @with_seed()
 def test_quantize_model():
+    if is_test_for_mkldnn():
+        dtype = 'uint8'
```

Comment:
Why different dtypes? the test is written to target int8, so why test uint8 now?

Reply:
Currently MKLDNN quantization only support uint8 as input data so we made separate logic for mkldnn.